[2023-12-19T04:08:06.381+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: steal_etl_storeToPostgres.steal_ETL manual__2023-12-19T04:08:05.297995+00:00 [queued]>
[2023-12-19T04:08:06.391+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: steal_etl_storeToPostgres.steal_ETL manual__2023-12-19T04:08:05.297995+00:00 [queued]>
[2023-12-19T04:08:06.392+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-19T04:08:06.404+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): steal_ETL> on 2023-12-19 04:08:05.297995+00:00
[2023-12-19T04:08:06.414+0000] {standard_task_runner.py:57} INFO - Started process 1038 to run task
[2023-12-19T04:08:06.416+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'steal_etl_storeToPostgres', 'steal_ETL', 'manual__2023-12-19T04:08:05.297995+00:00', '--job-id', '353', '--raw', '--subdir', 'DAGS_FOLDER/steal_etl_storeToPostgres.py', '--cfg-path', '/tmp/tmpew72c06l']
[2023-12-19T04:08:06.419+0000] {standard_task_runner.py:85} INFO - Job 353: Subtask steal_ETL
[2023-12-19T04:08:06.458+0000] {task_command.py:416} INFO - Running <TaskInstance: steal_etl_storeToPostgres.steal_ETL manual__2023-12-19T04:08:05.297995+00:00 [running]> on host f97daee24bd9
[2023-12-19T04:08:06.666+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='steal_etl_storeToPostgres' AIRFLOW_CTX_TASK_ID='steal_ETL' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T04:08:05.297995+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-19T04:08:05.297995+00:00'
[2023-12-19T04:08:06.691+0000] {warnings.py:109} WARNING - /opt/***/dags/steal_etl_storeToPostgres.py:54: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc.drop('_id', axis=1, inplace=True)

[2023-12-19T04:08:06.692+0000] {warnings.py:109} WARNING - /opt/***/dags/steal_etl_storeToPostgres.py:55: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc.rename(columns=newcolumns, inplace=True)

[2023-12-19T04:08:06.705+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-19T04:08:06.714+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=steal_etl_storeToPostgres, task_id=steal_ETL, execution_date=20231219T040805, start_date=20231219T040806, end_date=20231219T040806
[2023-12-19T04:08:06.750+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-19T04:08:06.893+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
